database/seeders/FrontendSeeder.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Seeders;
+
+use Blumilk\Meetup\Core\Models\Meetup;
+use Blumilk\Meetup\Core\Models\Organization;
+use Blumilk\Meetup\Core\Models\Speaker;
+use Blumilk\Meetup\Core\Models\User;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\Hash;
+
+class FrontendSeeder extends Seeder
+{
+    public function run(): void
+    {
+        $user = User::factory([
+            "name" => "Admin",
+            "email" => "admin@example.com",
+            "password" => Hash::make("password"),
+            "email_verified_at" => Carbon::createFromDate(2022, 01, 01),
+        ])
+            ->create();
+        Organization::factory(10)
+            ->create();
+        Speaker::factory(10)
+            ->create();
+        $speakers = Speaker::all();
+        $organization = Organization::all();
+        foreach ($speakers as $speaker) {
+            Meetup::factory(10)
+                ->create([
+                    "user_id" => $user,
+                    "speaker_id" => $speakers->random(),
```

Comment:
Are you sure that `random()` will return ID and not a Speaker instance?

Reply:
yes and I tested this function for run seeders and random return random object speakers for collection

Reply:
So I don't get it. You're asasigning an object to speaker id and it's okay?

Reply:
Speakers-> random return object sppeaker a laravel for this query assign object for relationship